_posts/2020-10-01-torch-network-from-scratch/torch_network_from_scratch.Rmd

```diff
@@ -0,0 +1,1076 @@
+---
+title: "Getting familiar with torch tensors"
+description: > 
+ In this first installment in a mini-series of four, we present the main things you will want to know about torch tensors. As an illustrative example, we'll code a simple neural network from scratch.
```

Comment:
```suggestion
 In this first installment of a four-part miniseries, we present the main things you will want to know about torch tensors. As an illustrative example, we'll code a simple neural network from scratch.
```

Reply:
Nothing technically wrong with the original, having an: "n-part miniseries" is just more typical (and thus a bit easier to understand).